common/build/eslint-config-fluid/minimal.js

```diff
@@ -358,6 +356,23 @@ module.exports = {
 				},
 			},
 		},
+		{
+			// Rules only for React files
+			files: ["*.jsx", "*.tsx"],
```

Comment:
Is it worth adding a printed config for the settings that apply to react files?

Reply:
Yeah, that sounds like a good idea.